### PR TITLE
US-7 Inventory GUI

### PR DESCRIPTION
**Pull request by DiachukVlad**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -62,3 +62,8 @@ crashlytics-build.properties
 
 .DS_Store
 	
+/.idea/.idea.UnityStrategy/.idea/encodings.xml
+/.idea/.idea.UnityStrategy/.idea/.gitignore
+/.idea/.idea.UnityStrategy/.idea/indexLayout.xml
+/.idea/.idea.UnityStrategy/.idea/riderModule.iml
```
Comment (DiachukVlad) [inline .gitignore:68]: Test comment

**Review by taras1808 — CHANGES_REQUESTED** [review]

File: Assets/Scripts/InventoryUI.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InventoryUI : MonoBehaviour
```
[the remark this answers is not in the document]
Comment (DiachukVlad) [reply Assets/Scripts/InventoryUI.cs:6]: done

File: Assets/Scripts/MouseLook.cs [hunk 3]
```diff
@@ -9,6 +9,8 @@ public class MouseLook : MonoBehaviour
 
     public Transform playerBody;
 
+    public GameObject inventoryUI;
```
[the remark this answers is not in the document]
Comment (DiachukVlad) [reply Assets/Scripts/MouseLook.cs:12]: Done

File: Assets/Scripts/MouseLook.cs [hunk 4]
```diff
@@ -18,6 +20,8 @@ void Start()
 
     void Update()
     {
+        if (inventoryUI.GetComponent<InventoryUI>().showingInventory) return;
```
[the remark this answers is not in the document]
Comment (DiachukVlad) [reply Assets/Scripts/MouseLook.cs:23]: done

**Review by taras1808 — APPROVED** [review]

**Pull request merged**